docs/configuration-reference/components/cert-manager.md

```diff
@@ -23,7 +23,6 @@ cert-manager component configuration example:
 component "cert-manager" {
   email = "example@example.com"
   namespace = "cert-manager"
```

Comment:
Unrelated, but we could fix formatting here.
```suggestion
  email     = "example@example.com"
  namespace = "cert-manager"
```

Reply:
Fixed!